Automattic/jetpack
# IDC: old code cleanup

Old code cleanup

## Proposed changes:

* Deleting previously deprecated methods.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create a test site and run standard IDC tests. Everything should work
* To test, you can use two JN sites and the Jetpack Debug tools. Use the tool to swap blog id on one of the sites with the ide from the other one. Safe mode should appear and all options should work (start fresh or migrate)
